controllers/api.articles.comments.controller.js

```diff
@@ -0,0 +1,23 @@
+const {
+  fetchArticlesComments,
+  fetchArticleById,
+} = require("../models/api.articles.comments.model");
+
+exports.getArticlesComments = (req, res, next) => {
+  const article_id = req.params.article_id;
+  if (isNaN(article_id)) {
```

Comment:
Postgres should throw an error for you if article ID is not a number when you try and execute your query. See if you can get hold of the error code and use it in your postgresql error handler instead of checking here, then you don't need to repeat similar logic in your other models 😄